Arevyn24 assignment 2

sshurie opened the pull request:
![memes-its-something](https://cloud.githubusercontent.com/assets/12241631/9701782/995a9aa4-540a-11e5-8ec7-a59ee23536c1.jpg)
Still need to complete 2 menu options

sshurie committed:
Initial Upload

Basic menu laid out, with three options added as well as the exit
command.

sshurie committed:
Fixed an issue caught in #3's if  statement

sshurie committed:
Stuff & Things

Added filler statements for input options 4 and 5

steve-plas commented:
Very nice clean code. I liked your use of the for loop and while loops!

sshurie commented:
Thank you! I still have a bit of work to do though. Your project looks awesome!

ninjapanzer commented:
Don't forget if you go to the `Files changed` tab it provides better context to apply comments inline with the code when referencing a structure or idea.

ninjapanzer commented on Assignment2.java:
```diff
@@ -0,0 +1,125 @@
+       }
+       
+       
+        while (bool == true) {
```
I agree with @ztmurphy21 that I appreciate you using future material. It helps me gauge acceptance of the future material and helps me plan how much time we will spend on it.

steve-plas commented:
@Arevyn24 Thank you! @ninjapanzer I forgot that i could do that, i will use that from now on lol

srtinkey commented on Assignment2.java:
```diff
@@ -0,0 +1,125 @@
+        }
+        System.out.println("Goodbye!");
+    };
+};
```
Nice! I like your menu items array and loop. I think it could use a few more comments, though.

sshurie replied:
Thanks for the input! I added more comments in a new commit, hope it looks a little better.

sshurie committed:
Switch Statement  and Slot Machine

-Change the main menu format from an if-else statement to a switch.
-Added menu option 4, the Slot Machine Simulator
-Added more comments

zzordo commented on Assignment2.java:
```diff
@@ -0,0 +1,249 @@
+        "Slot Machine Simulator","...Under Construction...", "Exit"
+       };
+       
+       // DISPLAY MENU
```
Good work commenting your code.  I'm not sure I see where the menu prints to the screen though.  This program has been very well thought out though!

_EDIT_ I see the for loop that prints the menuItems string array now.  Good idea.
